index.d.ts

```
@@ -0,0 +1,119 @@
+*/
+export function isEqual(ip1: string, ip2: string): boolean;
+
+/**
+* Convert an IP string into a buffer.
+*/
+export function toBuffer(ip: string, buffer?: Buffer, offset?: number): Buffer;
+
+/**
+* Convert an IP buffer into a string.
+*/
+export function toString(ip: Buffer, offset?: number, length?: number): string;
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)